rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy import (
+    Column,
+    Enum,
+    ForeignKey,
+    Integer,
+    String
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+
+ModelBase = declarative_base()
+
+
+class Game(ModelBase):
+    __tablename__ = 'game'
+
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    rounds_limit = Column(Integer)
+    winner_id = Column(Integer, ForeignKey('user.id'), nullable=False)
+
+    player_1 = relationship('User', foreign_keys=[player_1_id])
+    player_2 = relationship('User', foreign_keys=[player_2_id])
+    winner = relationship('User', foreign_keys=[winner_id])
+
+    rounds_remaining = relationship('Round', foreign_keys='Round.id')
+    current_round = relationship('Round', foreign_keys='Round.id')
+
+
+class Round(ModelBase):
+    __tablename__ = 'round'
+
+    id = Column(Integer, primary_key=True)
+    game_id = Column(Integer, ForeignKey('game.id'))
+    winning_choice_id = Column(Integer, ForeignKey('choice.id'), nullable=False)
```

Comment:
The terminology here is what's confusing things, I think :)

`NULL` is a value (of sorts) represents "no value". If a column is NULLable, it means it's allowed to hold cells which have no value- NULL. Contrastingly, if a column is NOT NULLable, no cells in that column are allowed to be NULL (i.e. everything must have a value, and that value cannot be `NULL`).

To break it down more simply:
- If a column should be able to contain `NULL` values, then the column is "NULL-able", so `nullable=True`.
- If a column should _not_ be able to contain `NULL` values, then the column is "NOT NULL-able", so `nullable=False`.